x-pack/test/security_solution_cypress/cypress/e2e/explore/users/user_risk_tab.cy.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { tag } from '../../../tags';
+
+import { login, visitUserDetailsPage } from '../../../tasks/login';
+
+import { cleanKibana, waitForTableToLoad } from '../../../tasks/common';
+
+import { ALERTS_COUNT, ALERT_GRID_CELL } from '../../../screens/alerts';
+
+describe('User risk tab', { tags: [tag.ESS, tag.SERVERLESS] }, () => {
+  before(() => {
+    cleanKibana();
+    cy.task('esArchiverLoad', 'risk_entities');
+    cy.task('esArchiverLoad', 'query_alert');
+  });
+
+  beforeEach(() => {
+    login();
+  });
+
+  after(() => {
+    cy.task('esArchiverUnload', 'risk_entities');
+    cy.task('esArchiverUnload', 'query_alert');
+  });
+
+  it('renders risk tab and alerts table', () => {
+    visitUserDetailsPage('user1');
+    cy.get('[data-test-subj="navigation-userRisk"]').click();
```

Comment:
The element should be wrapped into a screen constants and the action wrapped into a task method.